deadpool-lapin 0.13 compile error  on windows

error[E0277]: the trait bound `tokio_reactor_trait::Tokio: lapin::reactor::FullReactor` is not satisfied                                                                                                
  --> src\config.rs:92:31
   |
92 |                 .with_reactor(tokio_reactor_trait::Tokio),
   |                  ------------ ^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `reactor_trait::Reactor` is not implemented for `tokio_reactor_trait::Tokio`
   |                  |
   |                  required by a bound introduced by this call
   |
   = note: required for `tokio_reactor_trait::Tokio` to implement `lapin::reactor::FullReactor`
note: required by a bound in `ConnectionProperties::with_reactor`
  --> C:\Users\Yongge\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\lapin-3.0.2\src\connection_properties.rs:48:28
   |
48 |     pub fn with_reactor<R: FullReactor + Send + Sync + 'static>(mut self, reactor: R) -> Self {
   |                            ^^^^^^^^^^^ required by this bound in `ConnectionProperties::with_reactor`

--- a/crates/deadpool-lapin/Cargo.toml
+++ b/crates/deadpool-lapin/Cargo.toml
@@ -53,7 +53,7 @@ serde = { package = "serde", version = "1.0.103", features = [
     "derive",
 ], optional = true }
 tokio-executor-trait = { version = "2.1.0", optional = true }
-tokio-reactor-trait = { version = "2", optional = true }
+tokio-reactor-trait = { version = "3", optional = true }
 
 [dev-dependencies]
 config = { version = "0.15", features = ["json"] }

--- a/crates/deadpool-lapin/src/config.rs
+++ b/crates/deadpool-lapin/src/config.rs
@@ -87,9 +87,11 @@ impl Config {
         let conn_props = match runtime {
             None => conn_props,
             #[cfg(feature = "rt_tokio_1")]
-            Some(Runtime::Tokio1) => conn_props
-                .with_executor(tokio_executor_trait::Tokio::current())
-                .with_reactor(tokio_reactor_trait::Tokio),
+            Some(Runtime::Tokio1) => {
+                #[cfg(not(windows))]
+                let conn_props = conn_props.with_reactor(tokio_reactor_trait::Tokio);
+                conn_props.with_executor(tokio_executor_trait::Tokio::current())
+            }
             #[cfg(feature = "rt_async-std_1")]
             Some(Runtime::AsyncStd1) => conn_props
                 .with_executor(async_executor_trait::AsyncStd)

--- a/crates/deadpool-lapin/src/config.rs
+++ b/crates/deadpool-lapin/src/config.rs
@@ -89,7 +89,7 @@ impl Config {
             #[cfg(feature = "rt_tokio_1")]
             Some(Runtime::Tokio1) => {
                 #[cfg(not(windows))]
-                let conn_props = conn_props.with_reactor(tokio_reactor_trait::Tokio);
+                let conn_props = conn_props.with_reactor(tokio_reactor_trait::Tokio::current());
                 conn_props.with_executor(tokio_executor_trait::Tokio::current())
             }
             #[cfg(feature = "rt_async-std_1")]

